themes/classic/_dev/css/components/featuredproducts.scss

```diff
@@ -3,14 +3,48 @@ $product-description-height: 70px;
 
 #products,
 .featured-products,
-.product-accessories,
-.product-miniature {
+.product-accessories {
   .products {
     display: flex;
     flex-wrap: wrap;
     justify-content: flex-start;
   }
 
+  .products-section-title {
+    margin: 2.5rem 0;
+    font-weight: 500;
+    text-align: center;
+  }
+
+  .product {
+    padding: 0;
+
+    @at-root .page-index &, .page-search & {
+      width: 25%;
+      min-width: 250px;
```

Comment:
Do we really need these fixed sizes? Not sure

Reply:
@NeOMakinG no, we don't I just want to implement new stuff not to rebuild whole listing template.